vhdbuilder/packer/components.json

```diff
@@ -132,22 +124,6 @@
       "amd64OnlyVersions": [],
       "multiArchVersions": [
         "v3.24.6"
-      ],
-      "prefetchOptimizations": [
```

Comment:
I think you need to update the fields in bumping

Reply:
drafting PR, thanks

Reply:
changes merged